test/test-listing-fei.spec.ts

```diff
@@ -0,0 +1,346 @@
+    expect(proposalState).to.be.equal(7);
+    const {
+      configuration: { data },
+      aTokenAddress,
+      stableDebtTokenAddress,
+      variableDebtTokenAddress,
+    } = await pool.getReserveData(TOKEN);
+    const poolData = parsePoolData(data);
+    expect(poolData).to.be.eql({
+      reserveFactor: RESERVE_FACTOR,
+      reserved: '0',
+      stableRateEnabled: '0',
+      borrowingEnabled: '1',
+      reserveFrozen: '0',
+      reserveActive: '1',
+      decimals: DECIMALS,
+      liquidityBonus: '0',
+      LiquidityThreshold: '0',
+      LTV: '0',
+    });
+
+    // preparing for tests.
+    aFei = (await ethers.getContractAt('IERC20', aTokenAddress, proposer)) as IERC20;
+    stableDebt = (await ethers.getContractAt('IERC20', stableDebtTokenAddress, proposer)) as IERC20;
+    variableDebt = (await ethers.getContractAt(
+      'IERC20',
+      variableDebtTokenAddress,
+      proposer
+    )) as IERC20;
+    await (await fei.connect(feiHolder).approve(pool.address, parseEther('200000'))).wait();
+    await (await aave.connect(proposer).approve(pool.address, parseEther('200000'))).wait();
+
+    // AAVE deposit by proposer
+    await (await pool.deposit(aave.address, parseEther('100'), proposer.address, 0)).wait();
+    // FEI deposit by ampl holder
+    const depositedAmount = parseEther('100').div(decimalMultiplier);
+    await (
+      await pool.connect(feiHolder).deposit(fei.address, depositedAmount, FEI_HOLDER, 0)
+    ).wait();
+    expect(await aFei.balanceOf(FEI_HOLDER)).to.gte(depositedAmount.sub(1));
+    expect(await aFei.balanceOf(FEI_HOLDER)).to.lte(depositedAmount.add(1));
+
+    // FEI holder not able to borrow DAI against FEI
+    await expect(
+      pool.connect(feiHolder).borrow(dai.address, parseEther('1'), 2, 0, FEI_HOLDER)
+    ).to.be.revertedWith(ERRORS.NO_COLLATERAL_BALANCE);
+
+    // proposer able to borrow FEI variable against AAVE
+    const borrowAmount = parseEther('10').div(decimalMultiplier);
+    await (
+      await pool.connect(proposer).borrow(fei.address, borrowAmount, 2, 0, proposer.address)
+    ).wait();
+    expect(await variableDebt.balanceOf(proposer.address)).to.be.equal(borrowAmount);
+
+    // proposer not able to borrow FEI stable against AAVE
+    await expect(
+      pool.borrow(fei.address, borrowAmount, 1, 0, proposer.address)
+    ).to.be.revertedWith(ERRORS.NO_STABLE_BORROW);
+    increaseTime(40000);
+
+    // proposer able to repay FEI variable
+    await (await fei.connect(proposer).approve(pool.address, parseEther('100000'))).wait();
+    await (await pool.repay(fei.address, MAX_UINT_AMOUNT, 2, proposer.address)).wait();
+    expect(await variableDebt.balanceOf(proposer.address)).to.be.equal(parseEther('0'));
+  });
+
+  it("Incentives active on borrowing", async () => {
+    // The admin + emissions manager can't be the same address due to proxy restrictions 
+    // so we change it
+    await (await incentivesController.connect(feiDao).changeAdmin(FEI_HOLDER)).wait();
```

Comment:
You need to use the contract interface of the Proxy.
You can use this code to easily load the right function ABI and execute the transaction:
```
    const adminProxyAbi = ['function changeAdmin(address newOwner)'];
    const adminProxyInterface = new ethers.utils.Interface(adminProxyAbi);
    const encodedChangeAdmin = adminProxyInterface.encodeFunctionData('changeAdmin', [FEI_HOLDER]);
    await (
      await feiDao.sendTransaction({ data: encodedChangeAdmin, to: incentivesController.address })
    ).wait();
 ```

Reply:
I added the changeAdmin function to the IAaveIncentivesController interface to make this code work, but your solution is more transparent so I'll change it